Assets/MRTK/Providers/OculusXRSDK/OculusXRSDKTouchController.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.XRSDK.Input
+{
+    [MixedRealityController(
+        SupportedControllerType.OculusTouch,
+        new[] { Handedness.Left, Handedness.Right },
+        "StandardAssets/Textures/OculusControllersTouch")]
+    public class OculusXRSDKTouchController : GenericXRSDKController
+    {
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        public OculusXRSDKTouchController(TrackingState trackingState, Handedness controllerHandedness,
+            IMixedRealityInputSource inputSource = null, MixedRealityInteractionMapping[] interactions = null)
+            : base(trackingState, controllerHandedness, inputSource, interactions)
+        {
+        }
+
+        /// <inheritdoc />
+        public override MixedRealityInteractionMapping[] DefaultLeftHandedInteractions => new[]
+        {
+            new MixedRealityInteractionMapping(0, "Spatial Pointer", AxisType.SixDof, DeviceInputType.SpatialPointer),
+            new MixedRealityInteractionMapping(1, "Axis1D.PrimaryIndexTrigger", AxisType.SingleAxis, DeviceInputType.Trigger, ControllerMappingLibrary.AXIS_9),
```

Comment:
You can remove all these axis and keycode data, since they aren't used on XRSDK.

```suggestion
            new MixedRealityInteractionMapping(1, "Axis1D.PrimaryIndexTrigger", AxisType.SingleAxis, DeviceInputType.Trigger),
```